pkg/app/piped/notifier/matcher.go

```diff
@@ -26,8 +26,6 @@ type matcher struct {
 	ignoreGroups map[string]struct{}
 	apps         map[string]struct{}
 	ignoreApps   map[string]struct{}
-	envs         map[string]struct{}
-	ignoreEnvs   map[string]struct{}
```

Comment:
Removing this, we will need another matcher on `Label` since some teams are using this to divide their notifications into different Slack channels.

Reply:
I see, can I add a note here and address it by separated PR, since this one is only about removing the existing Env.

Reply:
Sure. Please add a TODO for it and we should have it in `v0.26.0`.